nx-dev/nx-dev/lib/api.ts

```diff
@@ -17,29 +16,14 @@ export function loadDocumentsData(): Map<string, DocumentMetadata[]> {
   const map = new Map<string, DocumentMetadata[]>();
   map.set('latest', latestDocuments);
   map.set('previous', previousDocuments);
-  if (process.env.VERCEL_ENV !== 'production') {
-    map.set('preview', previewDocuments);
-  }
   return map;
 }
 
 export function loadVersionsData(): VersionMetadata[] {
-  const versions: VersionMetadata[] = archiveVersionsData;
-  if (process.env.VERCEL_ENV !== 'production') {
-    versions.push({
-      name: 'Preview',
-      id: 'preview',
-      release: 'preview',
-      path: 'preview',
-      default: false,
-      hidden: true,
-    });
-  }
-  return versions;
+  return archiveVersionsData;
 }
 
 export const documentsApi = new DocumentsApi({
-  previewRoot: join(process.env.NX_WORKSPACE_ROOT, 'docs'),
   archiveRoot: join(
```

Comment:
I believe we can remove `NX_WORKSPACE_ROOT` altogether somehow since everything is under `public`. Maybe in a follow-up PR.